src/Scope.ts

```diff
@@ -370,6 +393,8 @@ export class Scope {
                         ns.classStatements[statement.name.text.toLowerCase()] = statement;
                     } else if (isFunctionStatement(statement) && statement.name) {
                         ns.functionStatements[statement.name.text.toLowerCase()] = statement;
+                    } else if (isFunctionStatement(statement) && statement.name) {
```

Comment:
right now this is a duplicate of the two lines above. Is this supposed to be enum-related?

```ts
 } else if (isEnumStatement(statement) && statement.name) {
    ns.enumStatements[statement.name.toLowerCase()] = statement;
}
```

Reply:
oops - must have merged bombed this.